_src/reference/chains.json

```diff
@@ -0,0 +1,388 @@
+[
+  {
+    "name": "Ethereum Mainnet",
+    "chain": "ETH",
+    "icon": "ethereum",
+    "rpc": [
+      "https://mainnet.infura.io/v3/${INFURA_API_KEY}",
```

Comment:
How are we going to maintain this file? These things change over time. Do we need all of this information for our simple "add to wallet" feature? it feels like there may be unnecessary fields here.

Reply:
@StephenFluin 
"add to wallet" feature has become "Switch to network (network must be added to metamask if not exist) then add to wallet". This data is required to add a network to metamask.
Originally, we were automatically pulling the info from this [source](https://chainid.network/chains.json) that is maintained by [chainlist](https://github.com/ethereum-lists/chains). Security requested to get a copy and validate it manually. Hence, every time that we support a new EVM chain, we pull data relevant to that chain, validate it and add it to "chains.json"